test/trace/span_context_test.rb

```diff
@@ -119,5 +108,35 @@
       context1.root.must_be_same_as root_context
       root_context.root.must_be_same_as root_context
     end
+
+    it "captures the stack trace" do
+      span_context = OpenCensus::Trace::SpanContext.create_root
+      span = span_context.start_span "hello"
+      frame = span.instance_variable_get(:@stack_trace).first
```

Comment:
It's kinda a bummer we have to resort to reading the instance variable. In the future, we can ask the SpanBuilder to return its read-only version once that's available.